pkg/bootstrap/testdata/xdsproxy_golden.json

```diff
@@ -249,9 +249,11 @@
       "transport_api_version": "V3",
       "grpc_services": [
         {
+        
           "envoy_grpc": {
             "cluster_name": "xds-grpc"
           }
+        
```

Comment:
Why is this change here?

Reply:
This is unrelated. It looks like a previous change did not actually update using the standard mechanism of:
`REFRESH_GOLDEN=true go test ./pkg/bootstrap/...`
I can remove it, but it will need to be updated eventually anyway to make it easier to actually make changes to the template.

Reply:
I backed this change out, I'll submit seperately.